### PR TITLE
Update woocommerce-admin to 1.6.0-rc.3

### PR DESCRIPTION
**Pull request by timmyc**
Here is the _hopefully_ final woocommerce-admin package update for 4.6 RC1.

This build contains one additional change from the prior that removes the logic for preventing the OBW redirect from happening:

https://github.com/woocommerce/woocommerce-admin/pull/5230

### CODE REVIEW

**Comment by rodrigoprimo** [comment]
Thanks, @timmyc.

Just documenting that I cherry-picked @rrennick's commit to update the e2e tests to reflect the new setup wizard flow from #27821. Waiting to see if Travis will pass with this change.

**Review by rodrigoprimo — APPROVED** [review]
Looks good to me. I could confirm that the setup wizard is now working as expected. Just waiting for one more review and then this is good to go.

**Review by jonathansadowski — APPROVED** [review]
Tested this with the change that was merged from #27822 and verified that the onboarding wizard was shown for new installs, but that when updating the onboarding wizard isn't shown (as expected).

**Pull request merged**
